packages/web/utils/__tests__/compare.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { Dec, Int } from "@keplr-wallet/unit";
+
+import {
+  compareCommon,
+  compareDec,
+  compareDefinedMember,
+  CompareResult,
+} from "../compare";
+
+describe("compareCommon", () => {
```

Comment:
Please take a look at these [tests](https://github.com/osmosis-labs/osmosis-frontend/blob/stage/packages/web/utils/__tests__/number.ts#L11) that leverage jest in case. It makes it much easier and readable to assert the same function with multiple values.